tests/csm/rest/test_iam_users.py

```diff
@@ -2127,6 +2126,190 @@ def test_39217(self):
+        assert resp.status_code == HTTPStatus.CREATED, "IAM user creation failed"
+        uid = resp.json()["user_id"]
+        self.created_iam_users.add(uid)
+
+        self.log.info("Step-2: Creating IAM user with same name as tenant")
+        payload = self.csm_obj.iam_user_payload_rgw("valid")
+        payload.update({"tenant": uid})
+        resp = self.csm_obj.create_iam_user_rgw(payload)
+        self.log.info("Verify Response : %s", resp)
+        assert resp.status_code == HTTPStatus.CREATED, "Status code check failed"
+
+    @pytest.mark.csmrest
+    @pytest.mark.lc
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-38969')
+    def test_38969(self):
+        """
+        Verify PATCH iam user request for access key that belongs to another user.
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("Step-1: Creating IAM user 1")
+        payload = self.csm_obj.iam_user_payload_rgw("valid")
+        resp = self.csm_obj.create_iam_user_rgw(payload)
+        self.log.info("Verify Response : %s", resp)
+        assert resp.status_code == HTTPStatus.CREATED, "IAM user creation failed"
+        usr1 = resp.json()
+        self.created_iam_users.add(usr1["user_id"])
+
+        self.log.info("Step-2: Creating IAM user 2")
+        payload = self.csm_obj.iam_user_payload_rgw("valid")
+        resp = self.csm_obj.create_iam_user_rgw(payload)
+        self.log.info("Verify Response : %s", resp)
+        assert resp.status_code == HTTPStatus.CREATED, "IAM user creation failed"
+        usr2 = resp.json()
+        self.created_iam_users.add(usr2["user_id"])
+
+        self.log.info("Step-3: Edit IAM user with access key of user 1")
+        payload = {"access_key": usr1["keys"][0]["access_key"],
+                    "secret_key":usr1["keys"][0]["secret_key"]}
+        resp = self.csm_obj.modify_iam_user_rgw(usr2["user_id"], payload)
+        assert resp.status_code == HTTPStatus.CONFLICT, "PATCH status code check failed"
+        resp = resp.json()
+        if CSM_REST_CFG["msg_check"] == "enable":
+            err_code = self.csm_conf["test_38969"]["error_code"]
+            assert int(resp["error_code"]) == err_code, "Error code check failed"
+            msg_id = self.csm_conf["test_38969"]["message_id"]
+            assert resp["message_id"] == msg_id, "Message id check failed"
+            msg = self.rest_resp_conf[err_code][msg_id][0]
+            assert resp["message"] == msg, "Message check failed"
+
+        self.log.info("##### Test completed -  %s #####", test_case_name)
+
+
+    @pytest.mark.csmrest
+    @pytest.mark.lc
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-38918')
+    def test_38918(self):
+        """
+        Attempt to modify Access key, secret key while IO operations are ongoing with old key
```

Comment:
we may need to change this test summary as we are not doing modification while ios are ongoing.

Reply:
Done